src/cart/utils/crt_utils.c

```diff
@@ -422,7 +424,7 @@ crtu_dc_mgmt_net_cfg_setenv(const char *name)
 
 	if (rc != 0) {
 		D_ERROR("dc_get_attach_info() failed, rc=%d\n", rc);
-		D_GOTO(cleanup, rc = d_errno2der(errno));
+		D_GOTO(cleanup, rc);
```

Comment:
In case of error dc_get_attach_info does not set errno and the rc error code is already relevant.